### PR TITLE
Finished the structure of the  suggestions page

### PR DESCRIPTION
**Pull request by iremchabanne**
Is there any unnecessary code in the CSS section that can be cleaned?

### CODE REVIEW

File: client/src/pages/SuggestionsPage.module.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+body:has(.mainSuggestion) {
+  background-image: url(../assets/images/wave.png),
+    url(../assets/images/airplane.png);
+  background-repeat: no-repeat;
+  background-position: 95% 30%;
+}
+
+.mainSuggestion {
+  h1 {
```
Comment (nimser) [inline client/src/pages/SuggestionsPage.module.css]: Redundant imbrication (just target `.mainSuggestion h1` directly)

File: client/src/pages/SuggestionsPage.module.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  }
+}
+
+@media only screen and (max-width: 600px) {
+  body:has(.mainSuggestion) {
+    background-image: none;
+  }
+
+  .mainSuggestion {
+    h1 {
```
Comment (nimser) [inline client/src/pages/SuggestionsPage.module.css]: Redundant imbrication (just target `.mainSuggestion h1` directly)

**Review by nimser — APPROVED** [review]
Good work. I made a minor suggestion in comments - avoid unnecessary selector nesting when nesting only one selector.

Please get someone in your team to test/review your code (and approve this PR) so we can merge it.

**Review by Naimoudine — APPROVED** [review]
looks good to me

**Pull request merged**